ovn-tester/ovn_tester.py

```diff
@@ -178,41 +187,83 @@ def configure_tests(yaml, central_node, worker_nodes, global_cfg):
         mod = importlib.import_module(f'tests.{section}')
         class_name = ''.join(s.title() for s in section.split('_'))
         cls = getattr(mod, class_name)
-        tests.append(cls(yaml, central_node, worker_nodes, global_cfg))
+        tests.append(cls(yaml, central_nodes, worker_nodes, global_cfg))
     return tests
 
 
 def create_nodes(cluster_config, central, workers):
-    mgmt_net = cluster_config.node_net
-    mgmt_ip = mgmt_net.ip + 2
-    internal_net = cluster_config.internal_net
-    external_net = cluster_config.external_net
-    gw_net = cluster_config.gw_net
-    db_containers = (
-        ['ovn-central-1', 'ovn-central-2', 'ovn-central-3']
-        if cluster_config.clustered_db
-        else ['ovn-central']
-    )
+    node_az_conf = [
+        NodeConf(
```

Comment:
I find this quite hard to read.  Do we really need this extra config structure that we use in this single place?  In the loop where WorkerNode instances are created we can just use the index to determine what sub-chunk of the subnet we need to assign to that node.